app/Community/Enums/AwardType.php

```diff
@@ -53,4 +53,17 @@ public static function toString(int $awardType): string
             default => 'Invalid or deprecated award type',
         };
     }
+
+    public static function game(): array
+    {
+        return [
+            self::Mastery,
+            self::GameBeaten,
+        ];
+    }
+
+    public static function isGame(int $type): bool
+    {
+        return !in_array($type, static::game());
```

Comment:
should be `in_array`:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/3426f0b0-1567-4293-a78f-5d606e09091d)

Reply:
yup, tests didn't like that either - nice.
fixed in https://github.com/RetroAchievements/RAWeb/pull/1959/commits/327b3d67e53ff5fc26ae16b45d98f84c9d529669